tests/Rule/InEnumTest.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Validator\Tests\Rule;
+
+use InvalidArgumentException;
+use Yiisoft\Validator\Rule\InEnum;
+use Yiisoft\Validator\Rule\InEnumHandler;
+use Yiisoft\Validator\Tests\Rule\Base\DifferentRuleInHandlerTestTrait;
+use Yiisoft\Validator\Tests\Rule\Base\RuleTestCase;
+use Yiisoft\Validator\Tests\Rule\Base\RuleWithOptionsTestTrait;
+use Yiisoft\Validator\Tests\Rule\Base\SkipOnErrorTestTrait;
+use Yiisoft\Validator\Tests\Rule\Base\WhenTestTrait;
+use Yiisoft\Validator\Tests\Support\Data\Enum\BackedEnumStatus;
+use Yiisoft\Validator\Tests\Support\Data\Enum\EnumStatus;
+
+final class InEnumTest extends RuleTestCase
```

Comment:
Could you also add tests for `int` backed enums?

Reply:
Done.